process.py

```diff
@@ -0,0 +1,11 @@
+class Proces:
+	def processEncrypt(self,data,shift):
+		to_file = " "
+		NUMBER_26 = ord('z')

```

Comment:
@Nithinkrishna1 What is NUMBER_26? when I read this, I dont get what it is. Better naming is required. Say TOTAL_ALPHABET_COUNT or something similar. Read about magic numbers. http://c2.com/cgi/wiki?MagicNumber